qlib/contrib/report/analysis_model/analysis_model_performance.py

```diff
@@ -49,8 +74,17 @@ def _group_return(pred_label: pd.DataFrame = None, reverse: bool = False, N: int
     # Long-Short
     t_df["long-short"] = t_df["Group1"] - t_df["Group%d" % N]
 
-    # Long-Average
-    t_df["long-average"] = t_df["Group1"] - pred_label.groupby(level="datetime")["label"].mean()
+    # Long-benchmark
+    benchmark = kwargs.get("benchmark", "average")
+    if isinstance(benchmark, str) and benchmark == "average":
+        benchmark_name = benchmark
+        benchmark = pred_label.groupby(level="datetime")["label"].mean()
+    elif isinstance(benchmark, pd.Series):
+        benchmark_name = benchmark.name if bool(benchmark.name) else "benchmark"
+        benchmark = benchmark.reindex(t_df.index)
```

Comment:
There is a high possibility that the user-provided benchmark may not align with t_df.index. Perhaps we can add some common warnings in this case if the DataFrame doesn't have warning messages.

Reply:
I think it might not be necessary as it should be a common case especially for a global portfolio. Think about using CSI300 to benchmarking a portfolio which invests both A and H shares, for those mainland holidays which are not HK holidays.